docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -40,24 +40,43 @@ abstract class Client {
   // get all notifications for a subscription
   public abstract getNotificationHistory(params: {
     topic: string,
-  }): Promise<Record<number, NotifyNotificationRecord>>
+    // Default 10, max 50
+    limit?: number,
+    starting_after?: string,
+  }): Promise<{
+    notifications: Record<number, NotifyNotificationRecord>,
+    has_more: boolean,
+    has_more_unread: boolean,
+  }>
+
+  // get notification by ID
+  public abstract getNotification(params: {
+    topic: string,
+    id: string,
+  }): Promise<NotifyNotificationRecord>
+
+  // mark notification as read
+  public abstract readNotification(params: {
+    topic: string,
+    id: string,
+  }): Promise<NotifyNotificationRecord>
+
+  // returns how many notifications are unread
+  public abstract getUnreadNotificationsCount(params: {
+    topic: string,
+  }): Promise<number>
 
   // delete active subscription
   public abstract deleteSubscription(params: {
     topic: string,
   }): Promise<void>;
-  
-  // delete notification
-  public abstract deleteNotification(params: {
-    id: number,
-  }): Promise<void>;
```

Comment:
Why are we getting rid of deletion?

Reply:
Because anything that was deleted would be restored via server-side history. Deletion is something that we aren't implementing server-side.

Reply:
I'm fine for now, however, I think we might benefit from deletion later on

Reply:
> I think we might benefit from deletion later on

Agree it's a possibility yeah, we'll drive this from developer feedback though!